src/core/lib/address_utils/sockaddr_utils.cc

```diff
@@ -41,7 +41,7 @@
 #include <sys/un.h>
 #endif
 
-std::string grpc_sockaddr_to_uri_unix_if_possible(
+static std::string grpc_sockaddr_to_uri_unix_if_possible(
```

Comment:
Could just move the function definition up here to avoid the forward declaration.

Reply:
True... I thought the file read a little nicer with the ifdefs towards the end away from the main code.
Will change it.